src/IconButton/IconButton.js

```diff
@@ -107,9 +110,14 @@ export default class IconButton extends Component {
         }, className)}
         centerRipple
         keyboardFocusedClassName={classes.keyboardFocused}
+        disabled={disabled}
         {...other}
       >
-        <span className={classes.label}>
+        <span
+          className={classNames(classes.label, {
+            [classes.disabled]: disabled,
```

Comment:
Why not applying the class on the `ButtonBase`?
That's the approach used on the `Button`. I believe it's for making overrides simpler for end-users.